04-swap-protocol.md

```diff
@@ -1,6 +1,8 @@
 # BOLD #4: Swap Protocol
 
-Once an order match is found on the taker order book, a swap procedure is expected to execute. The following is the protocol happy flow: 
+## Overview
+
+Once an order match is found in the taker’s order book, a swap procedure should execute. The following is the protocol happy flow: 
```

Comment:
, the swap protocol should be initiated. The current swap protocol assumes taker and maker to be connected via a payment channel network (e.g. the [Lightning](http://lightning.network/) or [Raiden Network](https://raiden.network/)) with sufficient balance for the swap available. The following is the swap protocol's "happy" flow: